cmd/template/cluster/flag.go

```diff
@@ -93,6 +109,7 @@ type flag struct {
 
 func (f *flag) Init(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&f.Provider, flagProvider, "", "Installation infrastructure provider.")
+	cmd.Flags().StringVar(&f.TplType, flagTplType, "raw", "Type of the template. Could be one of: raw, appcr. If application, cluster will be templated as an App CR. (default raw)")
```

Comment:
```suggestion
	cmd.Flags().StringVar(&f.TplType, flagTplType, "raw", "Type of the template. Could be one of: raw, app. If application, cluster will be templated as an App CR. (default raw)")
```

We use apps in our docs already. Let's see what sig UX says later 😅 but I prefer just `app`.

Reply:
I agree `app` looks better 😄